supplier/overview.md

```diff
@@ -16,14 +16,17 @@ As a supplier you should [Send an order response](send-order-response/) to the b
 
 Additionally you can [Attach a document to an order response](send-order-response/attach-document.md).
 
-## Step 3. Ship goods
+## Optional step 3. Send goods logistical status and eta in order response
 
-As a supplier you can [Ship goods](ship-goods.md).
+{% hint style="warning" %}
+This feature is planned and API and documentation may change.
+{% endhint %}
+
+TO BE DOCUMENTED using order line delivery schedule status and eta.
```

Comment:
```suggestion
```

imo; it's best to not leave "TO BE DONE"-like pointers in public documentation: this will remind customers more often than it will remind us ;)  
If this needs to be done quick, let's make a ticket for it instead.